examples/use_with_pymc.ipynb

```diff
@@ -273,16 +175,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "id": "a1de0d56",
    "metadata": {},
    "outputs": [],
    "source": [
+    "from pymc.sampling_jax import get_jaxified_graph\n",
+    "\n",
     "rvs = [rv.name for rv in model.value_vars]\n",
     "init_position_dict = model.compute_initial_point()\n",
     "init_position = [init_position_dict[rv] for rv in rvs]\n",
     "\n",
-    "logprob_fn = pm.sampling_jax.get_jaxified_logp(model)"
+    "def get_jaxified_logp(model):\n",
```

Comment:
Why not reuse the available `get_jaxified_logp` which is easier to use (and remember)?

Reply:
`get_jaxified_logp` returns the negation of the function blackjax is expecting. There is some refactoring that needs to happen on the pymc side but this is the cleanest way to make this notebook correct.